src/music.c

```diff
@@ -75,6 +75,8 @@ int distance;
                 npc_awakens();
             /* may scare some monsters -- waiting monsters excluded */
             if (!unique_corpstat(mtmp->data)
+                /* TNNT: treat devteam like unique mons when waiting */
```

Comment:
Can this get a 3.7 TODO to remove this code when we have shiny new lua `waiting=1`?